WORKSPACE

```diff
@@ -61,9 +54,10 @@ http_archive(
     ],
 )
 
-# START: Upstream TensorFlow dependencies
-# TensorFlow build depends on these dependencies.
-# Needs to be in-sync with TensorFlow sources.
+# The remainder of this file is derived from, and should be kept in sync with,
+# the TensorFlow WORKSPACE file:
+# https://github.com/tensorflow/tensorflow/blob/master/WORKSPACE
```

Comment:
The new `WORKSPACE` file will be easy to update with new TF versions, because you can basically paste in the TF `WORKSPACE` file below this line and apply the replacement `//tensorflow..` -> `@org_tensorflow//tensorflow...`.